app/facades/quotable_facade.rb

```diff
@@ -0,0 +1,11 @@
+class QuotableFacade
+  def self.search_quotes(keyword)
+    json = QuotableService.quotes_search_data(keyword)
+    # require 'pry', binding.pry
+    { total: json[:totalCount], quotes: json[:results].map do |quote_data|
```

Comment:
This is a great start, but we want to create a PORO that handles accessing the hashed data and turning them into objects for our view to use. See if you can get that implemented.